README.md

```diff
@@ -130,8 +134,9 @@ jobs:
           coverity_url: ${{ secrets.COVERITY_URL }}
```

Comment:
url is referred from vars not secrets in chuck's example. just fyi

Reply:
it's on users, they can configure accordingly